src/server/saved_objects/schema/schema.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+interface UIExportsSavedObjectTypeSchema {
```

Comment:
question: Why include `UIExports` in the name for these things? Isn't that just the mechanism in which they are consumed?

Reply:
We don't have to, I was attempting to differentiate between the `SavedObjectsSchema` itself and the data that we're receiving in the constructor for the `SavedObjectsSchema`, how about `SavedObjectsSchemaDefinition` instead of `UIExportsSavedObjectTypeSchema` or is there another name that you'd prefer? I'm still getting used to the inversion of control that typescript supports with regard to interfaces and their affect on naming...

Reply:
`SavedObjectsSchemaDefinition` works for me.